pipeline_dash/viz/dash/components/jobs_pipeline_fig.py

```diff
@@ -37,20 +40,30 @@ def scale_font_size(scale: float) -> float:
     return min(max(int(14 * scale), 6), 20)
 
 
-def size_traces(
-    scale: float, node_trace: go.Scatter, edge_traces: Dict[str, go.Scatter], annotations: List[go.layout.Annotation]
-):
+def size_traces(scale: float, node_trace: go.Scatter, edge_traces: Dict[str, go.Scatter]):
     node_trace.marker.size = max(node_trace.marker.size * scale, 2)
     for et in edge_traces.values():
         et.line.width = max(et.line.width * scale, 0.5)
+
+
+def size_annotations(scale: float, annotations: tuple[go.layout.Annotation, ...]):
```

Comment:
`tuple` here adds some weight to my previous comment about `Dict` vs `dict`. I'd recommend uniformity regardless of which direction you go